addons/mail/static/src/models/emoji_text_field_view.js

```diff
@@ -0,0 +1,97 @@
+/** @odoo-module **/
+
+import { attr, clear, one, Model} from '@mail/model';
+
+/**
+ * Watered-down version of composer.js
+ * used to instanciate an emoji picker popover
+ */
+Model({
+    name: 'EmojiTextFieldView',
+    identifyingMode: 'xor',
```

Comment:
just curious, what does it do ?

Reply:
I'm pretty sure it's so that if you have multiple fields that are marked 'identifying', you can only set 1 of them to use as a key to recover the related record. Whereas if I set it to 'and' I would need to set all of them and all of their values would be aggrgated into 1 key. 

So I might not need to set it since I only have 1 identifying field actually.